proxy/nginx/lualib/router.lua

```diff
@@ -20,7 +19,7 @@ local res, err = red:connect(redis_host, redis_port)
 
 -- Return if could not connect to Redis
 if not res then
-    return
+    return ngx.exit(ngx.HTTP_BAD_GATEWAY)
```

Comment:
These changes are for avoiding errors "[error] 19#19: *2 fallback could not be resolved (3: Host not found)"